frontend/ResourcesPage.css

```
@@ -0,0 +1,127 @@
+.resources-page {
+  padding: 20px;
+  font-family: "Poppins", sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)